tests/System.Azure.Experimental.Tests/System.Azure.Experimental.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```

Comment:
Out other projects don't show in text explorer. Only this one and System.Numerics.Tensors do show, and so I copied what's in Tensors.